android-components/components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -192,11 +192,14 @@ interface AwesomeBar {
      * @property providers The list of [SuggestionProvider]s in this group.
      * @property title An optional title for this group. The title may be rendered by an AwesomeBar
      * implementation.
+     * @property priority An optional priority for this group. Decides the order of this group
+     * in the AwesomeBar suggestions.
      * @property limit The maximum number of suggestions that will be shown in this group.
      * @property id A unique ID for this group (uses a generated UUID by default)
      */
     data class SuggestionProviderGroup(
         val providers: List<SuggestionProvider>,
+        var priority: Int? = 0,
```

Comment:
As we have a default value, is there any specific reason why this property should be null-able?

Reply:
Should we expand on how the priorities work? :)